### PR TITLE
build with 8.8.1

### PR DESCRIPTION
**Pull request by dten**

### CODE REVIEW

**Comment by jamesthompson** [comment]
It would be great to get this merged so stackage nightly picks it up soon. Thanks for doing the work @dten and thanks @tibbe for the excellent library and maintenance!

**Comment by 23Skidoo** [comment]
I'll cut a new release this week.

File: ekg-core.cabal [hunk 1]
```diff
@@ -34,7 +34,7 @@ library
 
   build-depends:
     ghc-prim < 0.6,
-    base >= 4.6 && < 4.13,
+    base >= 4.6 && < 4.14,
```
Comment (YoEight) [inline ekg-core.cabal]: Can we do `<5` instead?

**Comment by YoEight** [comment]
Hello @23Skidoo, could you cut a new release, please?

**Comment by 23Skidoo** [comment]
Sorry, completely forgot about this. Will do.

File: ekg-core.cabal (hunk 1 above)
Comment (23Skidoo) [reply ekg-core.cabal]: No, since we can't know that it's safe.

**Comment by YoEight** [comment]
Thank you @23Skidoo

File: .travis.yml [hunk 2]
```diff
@@ -26,6 +26,8 @@ before_cache:
   - rm -rfv $CABALHOME/packages/head.hackage
 matrix:
   include:
+    - compiler: ghc-8.8.1
```
Comment (ejconlon) [inline .travis.yml]: 8.8.2 now?

**Comment by YoEight** [comment]
Hello @23Skidoo, do you think you can cut a release this week? Thank you for your time.

**Comment by ejconlon** [comment]
@23Skidoo Friendly ping for a new release! I also have a few open PRs on the subprojects for this same migration. If you need maintenance help I would be happy to pitch in.

**Comment by TravisWhitaker** [comment]
Who is the current maintainer of this package, and what is necessary to get this merged?

**Comment by YoEight** [comment]
I start to worry about @23Skidoo, hope everything is ok.

**Comment by 23Skidoo** [comment]
Published `ekg-core-0.1.1.7` on Hackage. Also, @ejconlon is now an additional maintainer.

**Comment by ejconlon** [comment]
Thank you! I am working on getting it back into Stackage nightlies now.

**Pull request merged**